project/project.yml

```diff
@@ -22,3 +22,13 @@ sites:
         database: afbini
         status: development
         default: false
+    hseni:
+        name: 'hseni'
```

Comment:
You could probably change this to Health and safety Executive Northern Ireland but looks good.